lib/stub_content_metadata_parser.rb

```diff
@@ -6,6 +6,8 @@ def gem_content_metadata_style
       :simple_book
     elsif stub_object_type.include?('map')
       :map
+    elsif stub_object_type.casecmp('3d').zero? # just in case it comes in as 3D...
```

Comment:
I'm wondering if it would be better to have one valid way of specifying this in the data and validate that the data is correct rather than have to add many conditionals to our code.

Reply:
Indeed, it may be useful to rethink how this is done (though it would involve tracing back how the stub content metadata is generated and potentially updating there too).  For example, I believe the goobi system may generate stub content metadata